### ISSUE TITLE
Averages Chart: clean up t-statistics (only pre- and post- comparisons are needed)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#168]
```
Averages Chart: clean up t-statistics (only pre- and post- comparisons are 
needed)
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 30 Jun 2015 at 3:35

**Comment by lixun910** [comment]
fix is applied in 1.7.49

**Comment by lixun910** [comment]
update fix is divide the results to 2 groups:  1 for space and 1 for time.

(p-val and T Stat) all floating point numbers truncated to 3 digits.

D.F. truncated to 1 digit.

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.85 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
